### ISSUE TITLE
v3 bug with tiled sprites

### ISSUE THREAD

**Issue by azjgard** [replit/kaboom#706]
Hello!

First of all, love Kaboom; I've had a lot of fun playing with it :) thanks so much for all of your work!

-------

After upgrading to the v3 beta, I'm struggling to get a sprite to render with `tiled: true`. Here is some code for a minimum reproduction:
```js
kaboom();

loadSprite("grass", "grass.png");

add([
  sprite("grass", {
    width: width(),
    height: height(),
    tiled: true,
  }),
  pos(0, 0),
]);
``` 

When executing the code above with kaboom v2.8, the image tiles across the screen, as expected:
<img width="1906" alt="image" src="https://user-images.githubusercontent.com/19500384/232965710-cee8657b-e021-48ff-a778-a61ddef4391a.png">

When switching to execute the same code with kaboom v3, tiling appears to completely break:
<img width="1905" alt="image" src="https://user-images.githubusercontent.com/19500384/232965857-f6260982-b877-4cee-94e4-cb02a7443537.png">

For each version of kaboom, I'm loading the library from the unpkg CDN links:
- https://unpkg.com/kaboom@2000.2.8/dist/kaboom.js
- https://unpkg.com/kaboom@next/dist/kaboom.js

I know there are a good number of breaking changes, but from looking through the v3 docs and source code, I _think_ this code should work, but let me know if there's something that I've missed.

Thanks in advance!

-----

In case it's helpful, here's the sprite!
<img width="263" alt="grass" src="https://user-images.githubusercontent.com/19500384/232966446-5c9c6725-b175-48f4-aa18-682082ad7d30.png">

**Comment by slmjkdbtl** [comment]
Fixed in [3000.0.0-beta.5](https://github.com/replit/kaboom/commit/97530f1c34a1ba2d46d2b867dbc7ce5224855b2c)

**Comment by azjgard** [comment]
Thank you!
